test/code-build-test.js

```diff
@@ -144,6 +155,24 @@ describe("githubInputs", () => {
       "No source version could be evaluated."
     );
   });
+  it("can handle configuring update call-rate", () => {
+    process.env[`INPUT_PROJECT-NAME`] = projectName;
+    process.env[`INPUT_UPDATE-INTERVAL`] = updateInterval;
+    process.env[`INPUT_UPDATE-BACK-OFF`] = updateBackOff;
+    process.env[`GITHUB_REPOSITORY`] = repoInfo;
+    process.env[`GITHUB_SHA`] = sha;
+
+    require("@actions/github").context.payload = {};
```

Comment:
this new test failed.

Can you try setting the context here? [sample](https://github.com/aws-actions/aws-codebuild-run-build/blob/master/test/code-build-test.js#L115)